server_tests/package.json

```diff
@@ -20,12 +20,11 @@
   "scripts": {
     "test": "./node_modules/.bin/mocha --reporter spec",
     "test-bubl": "SCARLET_TEST_HOST=192.168.0.100 SCARLET_TEST_PORT=80 SCARLET_TEST_WIFI=1 SCARLET_TEST_MODEL=bubl1 ./node_modules/.bin/mocha --reporter spec",
-    "lint": "./node_modules/.bin/standard"
+    "lint": "./node_modules/.bin/jsfmt --validate ./test"
```

Comment:
It no longer does the checks for standard, just jsfmt, since standard does something funky with tabs after a return statement. I am not sure why the Standards repo would need modification? You are referring to our internal Standards repo right?